pkg/rtc/participant.go

```diff
@@ -1314,30 +1314,26 @@ func (p *ParticipantImpl) configureReceiverDTX() {
 	}
 }
 
-func (p *ParticipantImpl) onStreamedTracksChange(paused map[string][]string, resumed map[string][]string) error {
-	if len(paused) == 0 && len(resumed) == 0 {
+func (p *ParticipantImpl) onStreamedTracksChange(update *sfu.StreamedTracksUpdate) error {
+	if len(update.Paused) == 0 && len(update.Resumed) == 0 {
 		return nil
 	}
 
 	streamedTracksUpdate := &livekit.StreamedTracksUpdate{}
-	if len(paused) != 0 {
-		for participantId, trackIds := range paused {
-			for _, trackId := range trackIds {
-				streamedTracksUpdate.Paused = append(streamedTracksUpdate.Paused, &livekit.StreamedTrack{
-					ParticipantSid: participantId,
-					TrackSid:       trackId,
-				})
-			}
+	if len(update.Paused) != 0 {
```

Comment:
@davidzhao I made `StreamedTracksUpdate` in `StreamAllocator` which is same as the protocol one. But, still had to do this loop as the types are different. Maybe there is a shortcut in golang to avoid this?

Reply:
check length is no need, for range in a nil array has no probelm

Reply:
Got it. Will clean it up